build/build-treemap.js

```diff
@@ -19,10 +18,15 @@ const {LH_ROOT} = require('../root.js');
  */
 function buildStrings() {
   const locales = require('../lighthouse-core/lib/i18n/locales.js');
-  const UIStrings = require(
-    // Prevent `tsc -p .` from evaluating util.js using core types, it is already typchecked by `tsc -p lighthouse-treemap`.
-    '' + '../lighthouse-treemap/app/src/util.js'
-  ).UIStrings;
+  // TODO(esmodules): use simple import when build/ is esm.
+  // const UIStrings = require(
+  //   // Prevent `tsc -p .` from evaluating util.js using core types, it is already typchecked by `tsc -p lighthouse-treemap`.
+  //   '' + '../lighthouse-treemap/app/src/util.js'
+  // ).UIStrings;
+  const {UIStrings} = eval(
+    fs.readFileSync(LH_ROOT + '/lighthouse-treemap/app/src/util.js', 'utf-8')
+      .replace('export ', '') + '\nmodule.exports = TreemapUtil;'
+  );
```

Comment:
this was a bit tricky to follow, wdyt about...

```suggestion
  const utilEsm = fs.readFileSync(LH_ROOT + '/lighthouse-treemap/app/src/util.js', 'utf-8');
  const utilCommonjs = `${utilEsm.replace('export ', '')}\nmodule.exports = TreemapUtil;`
  const {UIStrings} = eval(utilCommonjs);
```